jep/1/README.adoc

```diff
@@ -0,0 +1,754 @@
+
+Informational and Process JEPs may be updated over time to reflect changes
+to development practices and other details. The precise process followed in
+these cases will depend on the nature and purpose of the JEP being updated.
+
+Replaced::  [[replaced]]
+Final JEPs may eventually also be "Replaced" - superseded by a different JEP -
+rendering the original obsolete.
+This is intended for Informational JEPs, where version 2 of an API can replace version 1.
+
+==== Other JEP Outcomes
+
+Not all JEPs will be accepted and finalized.
+
+Rejected::  [[rejected]]
+A JEP can also be "Rejected". Perhaps after all is said and done it
+was not a good idea. It is still important to have a record of this
+fact.
+
+Rejected::  [[withdrawn]]
+The "Withdrawn" status is similar to "Rejected" - it means that the JEP author
+themselves has decided that the JEP is actually a bad idea,
+or agrees that a competing proposal is a better alternative.
+
+Deferred:: [[deferred]]
+A JEP can also be assigned a status of "Deferred". The JEP author or an
+editor can assign the JEP this status when no progress is being made
+on the JEP. Once a JEP is deferred, a JEP editor can re-assign it
+to draft status.
+
+Active:: [[active]]
+Some Informational and Process JEPs may also have a status of "Active" if they
+are ongoing and never meant to be completed per se. E.g. JEP 1 (this JEP).
+
+==== Updating JEP Status and Resolution
+
+Whenever a JEP status changes, the "Status" field in the JEP document must be updated.
+
+The possible paths of a JEP's status are as follows:
+
+.JEP Workflow
+image::workflow.png[JEP Workflow]
+
+When a JEP is Accepted, Rejected or Withdrawn,
+the "Resolution" header must be added with a link to the relevant post
+in the jenkinsci-dev@ mailing list archives.
+
+
+=== What belongs in a successful JEP?
+
+Each JEP should have the following parts:
+
+. **Metadata** - table containing metadata about the JEP, including the JEP
+  number, a short descriptive title, the names, and optionally the contact info
+  for each author, etc.
+. **Abstract** - short (200 word) description of the technical issue
+  being addressed.
+. **Specification** - The technical specification should describe the
+  syntax and semantics of any new feature. The specification should be
+  sufficiently detailed to allow new or existing Jenkins developers to
+  reasonably understand the scope/impact of an implementation.
+. **Motivation** - The motivation is critical for JEPs that want to
+  change Jenkins itself. It should clearly explain why the
+  existing code base is inadequate to address the
+  problem that the JEP solves. JEP submissions without sufficient
+  motivation may be rejected outright.
+. **Reasoning** - The reasoning describes why particular design
+  decisions were made. It should describe alternate designs that
+  were considered and related work, e.g. how the feature is supported
+  in other languages.
```

Comment:
"languages" comes from Python. In the case of Jenkins I would rather propose using generic "systems" or "solutions"